http://http://bluegrassrivals.com/forum/favicon.ico ==> HTTPConnectionPool(host='http', port=80): Max retries exceeded with url: //bluegrassrivals.com/forum/favicon.ico (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fc283a51d50>: Failed to establish a new connection: [Errno -2] Name or service not known'))
https://caringbridge.orghttp//www.caringbridge.org/favicon.ico ==> HTTPSConnectionPool(host='caringbridge.orghttp', port=443): Max retries exceeded with url: //www.caringbridge.org/favicon.ico (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fc283975150>: Failed to establish a new connection: [Errno -2] Name or service not known'))
http://http://bluegrassrivals.com/forum/favicon.ico ==> HTTPConnectionPool(host='http', port=80): Max retries exceeded with url: //bluegrassrivals.com/forum/favicon.ico (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7faff094f810>: Failed to establish a new connection: [Errno -2] Name or service not known'))
https://caringbridge.orghttp//www.caringbridge.org/favicon.ico ==> HTTPSConnectionPool(host='caringbridge.orghttp', port=443): Max retries exceeded with url: //www.caringbridge.org/favicon.ico (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7faff0958150>: Failed to establish a new connection: [Errno -2] Name or service not known'))
http://http://bluegrassrivals.com/forum/favicon.ico ==> HTTPConnectionPool(host='http', port=80): Max retries exceeded with url: //bluegrassrivals.com/forum/favicon.ico (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f7dcfb4edd0>: Failed to establish a new connection: [Errno -2] Name or service not known'))
https://caringbridge.orghttp//www.caringbridge.org/favicon.ico ==> HTTPSConnectionPool(host='caringbridge.orghttp', port=443): Max retries exceeded with url: //www.caringbridge.org/favicon.ico (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f7dcfa6cd50>: Failed to establish a new connection: [Errno -2] Name or service not known'))
https://nattyornotforum.nattyornot.com ==> HTTPSConnectionPool(host='nattyornotforum.nattyornot.com', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f7dd08978d0>: Failed to establish a new connection: [Errno -2] Name or service not known'))
http://http://bluegrassrivals.com/forum/favicon.ico ==> HTTPConnectionPool(host='http', port=80): Max retries exceeded with url: //bluegrassrivals.com/forum/favicon.ico (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fba0e64b890>: Failed to establish a new connection: [Errno -2] Name or service not known'))
https://caringbridge.orghttp//www.caringbridge.org/favicon.ico ==> HTTPSConnectionPool(host='caringbridge.orghttp', port=443): Max retries exceeded with url: //www.caringbridge.org/favicon.ico (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fba0e6500d0>: Failed to establish a new connection: [Errno -2] Name or service not known'))
https://wordpress ==> HTTPSConnectionPool(host='wordpress', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fe7f4499010>: Failed to establish a new connection: [Errno -2] Name or service not known'))
https://nattyornotforum.nattyornot.com ==> HTTPSConnectionPool(host='nattyornotforum.nattyornot.com', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fe7f80f24d0>: Failed to establish a new connection: [Errno -2] Name or service not known'))
http://http://bluegrassrivals.com/forum/favicon.ico ==> HTTPConnectionPool(host='http', port=80): Max retries exceeded with url: //bluegrassrivals.com/forum/favicon.ico (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fe7d4343450>: Failed to establish a new connection: [Errno -2] Name or service not known'))
https://rocketreach.cohttp//static.rocketreach.co/images/favicons/favicon-32x32.png?v=2020120 ==> HTTPSConnectionPool(host='rocketreach.cohttp', port=443): Max retries exceeded with url: //static.rocketreach.co/images/favicons/favicon-32x32.png?v=2020120 (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fe7b6c7c910>: Failed to establish a new connection: [Errno -2] Name or service not known'))
https://caringbridge.orghttp//www.caringbridge.org/favicon.ico ==> HTTPSConnectionPool(host='caringbridge.orghttp', port=443): Max retries exceeded with url: //www.caringbridge.org/favicon.ico (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fe7f78eccd0>: Failed to establish a new connection: [Errno -2] Name or service not known'))
https://7cups.comhttp//d37v7cqg82mgxu.cloudfront.net/img/favicon.png ==> HTTPSConnectionPool(host='7cups.comhttp', port=443): Max retries exceeded with url: //d37v7cqg82mgxu.cloudfront.net/img/favicon.png (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fe7f7406490>: Failed to establish a new connection: [Errno -2] Name or service not known'))
http://http://bluegrassrivals.com/forum/favicon.ico ==> HTTPConnectionPool(host='http', port=80): Max retries exceeded with url: //bluegrassrivals.com/forum/favicon.ico (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f510c6ebe50>: Failed to establish a new connection: [Errno -2] Name or service not known'))
http://http://bluegrassrivals.com/forum/favicon.ico ==> HTTPConnectionPool(host='http', port=80): Max retries exceeded with url: //bluegrassrivals.com/forum/favicon.ico (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f5964638490>: Failed to establish a new connection: [Errno -2] Name or service not known'))
http://http://bluegrassrivals.com/forum/favicon.ico ==> HTTPConnectionPool(host='http', port=80): Max retries exceeded with url: //bluegrassrivals.com/forum/favicon.ico (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f8b71648390>: Failed to establish a new connection: [Errno -2] Name or service not known'))
https://caringbridge.orghttp//www.caringbridge.org/favicon.ico ==> HTTPSConnectionPool(host='caringbridge.orghttp', port=443): Max retries exceeded with url: //www.caringbridge.org/favicon.ico (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f8b71899550>: Failed to establish a new connection: [Errno -2] Name or service not known'))
https://about.me ==> 'list' object has no attribute 'get'
https://any.do ==> 'list' object has no attribute 'get'
https://archive.org ==> 'list' object has no attribute 'get'
